src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -672,23 +707,39 @@ public static void createAndAddNewObjectMap(@NonNull Object id, List<Map<String,
         objectList.add(newObjectMap);
     }
 
+    public static Map<String, Object> getObjectsMapHavingClient(RestHighLevelClient client, String index, String id, String object) throws Exception {
```

Comment:
What is `object` here?  Can you rename with more informative variable?  Similar comment for L722.

Reply:
How about `getObjectsMap` instead of the `getObjectsMapHavingClient`?

Reply:
I guess I don't need getObjectsMapHavingClient at all? Maybe I delete it?

Reply:
If it's not needed by anything, sure!